Documentation/Upgrade/rook-upgrade.md

```diff
@@ -48,21 +49,25 @@ those releases.
     official releases. Builds from the master branch can have functionality changed or removed at any
     time without compatibility support and without prior notice.
 
-## Breaking changes in v1.14
+## Breaking changes in v1.15
 
-* The minimum supported version of Kubernetes is v1.25.
-    Upgrade to Kubernetes v1.25 or higher before upgrading Rook.
-* The Rook operator config `CSI_ENABLE_READ_AFFINITY` was removed. v1.13 clusters that have modified
-    this value to be `"true"` must set the option as desired in each CephCluster as documented
-    [here](https://rook.github.io/docs/rook/v1.14/CRDs/Cluster/ceph-cluster-crd/#csi-driver-options)
-    before upgrading to v1.14.
-* Rook is beginning the process of deprecating CSI network "holder" pods.
+* Rook has CSI network "holder" pods.
     If there are pods named `csi-*plugin-holder-*` in the Rook operator namespace, see the
     [detailed documentation](../CRDs/Cluster/network-providers.md#holder-pod-deprecation)
-    to disable them. This is optional for v1.14, but will be required in a future release.
-* In the operator helm chart, the images for the CSI driver are now specified with separate
-    `repository` and `tag` values. If the CSI images have been customized, convert them from the
-    `image` value to the separated `repository` and `tag` values.
+    to disable them. This deprecation process is required before upgrading to the future Rook v1.16.
+
+* Ceph COSI driver images have been updated. This impacts existing COSI Buckets, BucketClaims, and
+    BucketAccesses. Update existing clusters following the guide
+    [here](https://github.com/rook/rook/discussions/14297).
+
+* During CephBlockPool updates, Rook will now return an error if an invalid device class is
```

Comment:
I'd rather not refer to a PR that doesn't have clear actions. Let me get back to you on more specific text here...

Reply:
As discussed, this note will affect very few users. We will remove this from the upgrade guide and instead make sure it has a quality note in the release notes.